### ISSUE TITLE
JURA E6 - Coffee-Strength not according to the setting 

### ISSUE THREAD

**Issue by PortInnovator** [AlexxIT/Jura#19]
The E6 allows coffee-strength setting in the range from 1 to 8 (coffee-)beans. With the present integration you have to select the strength level between 3-10 to get 1-8 beans. Would be nice to have a configuration option for the respective JURA maschine type to get the right mapping.

**Comment by AlexxIT** [comment]
Show Bluetooth integration diagnostics

**Comment by PortInnovator** [comment]
Can you explain what's the relevance of BT in this case, please?
My BT setup (long range) is working properly with J.O.E.-App, Bluetti and Shelly BT devices.

**Comment by AlexxIT** [comment]
I need to know exact model number

**Comment by AlexxIT** [comment]
If you afraid to share Bluetooth info, manufacturer line of this device will be enough

**Comment by PortInnovator** [comment]
It's the E6 (EB) from 2020

**Comment by AlexxIT** [comment]
Company has more than 500 different models. Many of them has similar name. I need 5 digit number.

**Comment by PortInnovator** [comment]
Alright I got it from the machine label. It's the 15260 (Type 734)

**Comment by AlexxIT** [comment]
> Do they really have 500 different firmware variants?

No. They has mapping from digits to config. They has only 69 configs.
